modules/openfast-library/src/FAST_Solver.f90

```diff
@@ -989,6 +990,11 @@ SUBROUTINE SrvD_InputSolve( p_FAST, m_FAST, u_SrvD, y_ED, y_IfW, y_OpFM, y_BD, y
    u_SrvD%LSShftFys = y_ED%LSShftFys
    u_SrvD%LSShftFzs = y_ED%LSShftFzs
 
+
+   ! Set inputs here!!
+   u_SrvD%AllOutData = y_FAST%AllOutData
```

Comment:
Pass only the last index of `y_FAST%AllOutData` and make 1D.  Can drop `n_out` then.